frontend/src/constants.js

```diff
@@ -60,3 +60,28 @@ export const DEFAULT_POST_PROCESSOR = {
     size: 'size',
     date: 'date',
 };
+
+export const PRE_POPULATE_SCHEMA_ROWS = [
```

Comment:
consider uniformity with https://github.com/codalab/codalab-worksheets/blob/master/codalab/lib/worksheet_util.py#L575

Reply:
Thanks I updated the fields so it's the same as the default ones.

Reply:
Sounds good. Can we rename to `DEFAULT_SCHEMA_ROWS`?

Reply:
Also add a comment describing what this is used for.

Finally, add a comment here saying that it should be synced with https://github.com/codalab/codalab-worksheets/blob/master/codalab/lib/worksheet_util.py#L575, and a comment in https://github.com/codalab/codalab-worksheets/blob/master/codalab/lib/worksheet_util.py#L575 saying it has to be synced with this file.

Reply:
Thanks I have renamed the constant and updated the comments.